src/algorithms/spectral/logspectrum.cpp

```diff
@@ -24,68 +24,76 @@ using namespace std;
-const char* LogSpectrum::category = "Spectral";
-const char* LogSpectrum::description = DOC("This algorithm computes spectrum with logarithmically distributed frequency bins. "
-"This code is ported from NNLS Chroma [1, 2]."
-"This algorithm also returns a local tuning that is retrieved for input frame and a global tuning that is updated with a moving average.\n"
-"\n"
-"Note: As the algorithm uses moving averages that are updated every frame it should be reset before  "
-"processing a new audio file. To do this call reset() (or configure())\n"
-"\n"
-"References:\n"
-"  [1] Mauch, M., & Dixon, S. (2010, August). Approximate Note Transcription\n"
-"  for the Improved Identification of Difficult Chords. In ISMIR (pp. 135-140).\n"
-"  [2] Chordino and NNLS Chroma,\n"
-"  http://www.isophonics.net/nnls-chroma");
-
-void LogSpectrum::configure() {
+const char *LogSpectrum::name = "LogSpectrum";
+const char *LogSpectrum::category = "Spectral";
+const char *LogSpectrum::description = DOC("This algorithm computes spectrum with logarithmically distributed frequency bins. "
+                                           "This code is ported from NNLS Chroma [1, 2]."
+                                           "This algorithm also returns a local tuning that is retrieved for input frame and a global tuning that is updated with a moving average.\n"
+                                           "\n"
+                                           "Note: As the algorithm uses moving averages that are updated every frame it should be reset before  "
+                                           "processing a new audio file. To do this call reset() (or configure())\n"
+                                           "\n"
+                                           "References:\n"
+                                           "  [1] Mauch, M., & Dixon, S. (2010, August). Approximate Note Transcription\n"
+                                           "  for the Improved Identification of Difficult Chords. In ISMIR (pp. 135-140).\n"
+                                           "  [2] Chordino and NNLS Chroma,\n"
+                                           "  http://www.isophonics.net/nnls-chroma");
+
+void LogSpectrum::configure()
+{
   // Get parameters.
   _frameSize = parameter("frameSize").toInt();
   _sampleRate = parameter("sampleRate").toFloat();
   _rollon = parameter("rollOn").toFloat();
   _nBPS = parameter("binsPerSemitone").toInt();
-  _nOctave = 7;
-  _nNote = _nOctave * 12 * _nBPS + 2 * (_nBPS/2+1); // a core over all octaves, plus some overlap at top and bottom
+  _nOctaves = parameter("nOctaves").toInt();             //7;
+  _nNote = _nOctaves * 12 * _nBPS + 2 * (_nBPS / 2 + 1); // a core over all octaves, plus some overlap at top and bottom
   initialize();
 }
 
-
-void LogSpectrum::compute() {
-  const vector<Real>& const_spectrum = _spectrum.get();
+void LogSpectrum::compute()
+{
+  const vector<Real> &const_spectrum = _spectrum.get();
```

Comment:
Same for `&`, have a look at our current style (for consistency with the majority of the code).